src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -51,41 +51,74 @@ internal static bool IsNativeSupportEnabled(SentryUnityOptions options, IDiagnos
+                CopyFile(nativeBridgePath, Path.Combine(pathToProject, "Libraries", SentryPackageInfo.GetName(), SentryXcodeProject.BridgeName), logger);
+
+                using var sentryXcodeProject = SentryXcodeProject.Open(pathToProject);
+                sentryXcodeProject.AddSentryNativeBridge();
+            }
+            catch (Exception e)
+            {
+                logger.LogError("Failed to add the Sentry NoOp bridge to the output project.", e);
+            }
+        }
+
+        internal static void SetupSentry(SentryUnityOptions options,
+            SentryCliOptions? cliOptions,
+            IDiagnosticLogger logger,
+            string pathToProject)
+        {
             logger.LogInfo("Attempting to add Sentry to the Xcode project.");
 
             try
             {
-                // The Sentry.xcframework ends in '~' to stop Unity from trying to import the .frameworks.
-                // Otherwise, Unity tries to export those with the XCode build.
-                var frameworkPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins", "iOS", "Sentry.xcframework~"));
-                CopyFramework(frameworkPath, Path.Combine(pathToProject, "Frameworks", "Sentry.xcframework"), logger);
+                // The Sentry.xcframework ends in '~' to hide it from Unity. Otherwise, Unity tries to export it with the XCode build.
+                var frameworkPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins", "iOS", SentryXcodeProject.FrameworkName + "~"));
+                CopyFramework(frameworkPath, Path.Combine(pathToProject, "Frameworks", SentryXcodeProject.FrameworkName), logger);
 
-                var nativeBridgePath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins", "iOS", "SentryNativeBridge.m"));
-                CopyFile(nativeBridgePath, Path.Combine(pathToProject, "Libraries", SentryPackageInfo.GetName(), "SentryNativeBridge.m"), logger);
+                var nativeBridgePath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins", "iOS", SentryXcodeProject.BridgeName));
+                CopyFile(nativeBridgePath, Path.Combine(pathToProject, "Libraries", SentryPackageInfo.GetName(), SentryXcodeProject.BridgeName), logger);
 
                 using var sentryXcodeProject = SentryXcodeProject.Open(pathToProject);
                 sentryXcodeProject.AddSentryFramework();
                 sentryXcodeProject.AddSentryNativeBridge();
                 sentryXcodeProject.AddNativeOptions(options, NativeOptions.CreateFile);
                 sentryXcodeProject.AddSentryToMain(options);
 
-                if (cliOptions?.IsValid(logger, EditorUserBuildSettings.development) is true)
+                if (cliOptions != null && cliOptions.IsValid(logger, EditorUserBuildSettings.development))
```

Comment:
`cliOptions` is a Unity object. They don't "like" null-coalescing operators and create warnings.